### DISCUSSION TITLE
部分 OJ 的 VJudge 方法 - HeRaNO

### DISCUSSION THREAD

**Discussion by HeRaNO** [HeRaNO/HeRaNO.github.io#171] [Archive]
https://herano.xyz/2021/01/27/%E9%83%A8%E5%88%86-OJ-%E7%9A%84-VJudge-%E6%96%B9%E6%B3%95/ 

 主要讲的就是怎么搭（mo）VJudge（yu）

**Comment by memset0** [comment]
loj不是开的![IesrgfzB](https://user-images.githubusercontent.com/34177126/106619886-fd650080-65ab-11eb-84b3-be927fca51e2.png)嘛（（

**Comment by HeRaNO** [comment]
> @memset0
> loj不是开的![IesrgfzB](https://user-images.githubusercontent.com/34177126/106619886-fd650080-65ab-11eb-84b3-be927fca51e2.png)嘛（（

啊这，这项只有全站管理员可以改的。
